tests/build_test.sh

```diff
@@ -238,6 +238,18 @@ function test_kernel_build_menu_cross_compilation_flags()
   assertEquals "($LINENO)" "$expected_result" "$output"
 }
 
+function test_kernel_build_clean() {
+  local expected_result
+  local output
+
+  build_config=()
+  parse_configuration "$SAMPLES_DIR/build_no_log.config" build_config
```

Comment:
Please, refer to: https://kworkflow.org/content/codingstyle.html#string-concatenation